server/src/main/java/org/elasticsearch/snapshots/RepositoryIntegrityHealthIndicatorService.java

```diff
@@ -44,21 +53,43 @@ public class RepositoryIntegrityHealthIndicatorService implements HealthIndicato
 
     public static final String NAME = "repository_integrity";
 
-    public static final String HELP_URL = "https://ela.st/fix-repository-integrity";
```

Comment:
Out of curiosity why did you choose to inline this?

Reply:
I initially did that because I added the anchors to the links (and thought it would save some memory by not constructing the help URLs but inlining them), but I forgot to extract the constant after I removed the anchor. Good catch, thanks